packages/ai/core/util/detect-mimetype.ts

```diff
@@ -83,19 +85,48 @@ export const audioMimeTypeSignatures = [
   },
 ] as const;
 
+const stripID3 = (data: Uint8Array | string) => {
+  const bytes =
+    typeof data === 'string' ? convertBase64ToUint8Array(data) : data;
```

Comment:
This leads to unnecessary conversion of potentially large data entries. Can the edit in the base64 case happen w/o converting to UInt8Array?

Reply:
I did spend quite a while on this today, I couldn't get raw base64 stripping working. You can detect the existence of id3 tags by the prefix `SURz` but I think the remaining data between the prefix and the start of the raw mp3 frames are variable.

I'd love to know whether it's possible but yeah I couldn't get it working.